.github/workflows/test_functional.yml

```diff
@@ -130,6 +130,7 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y sqlite3 tree at
+          sudo apt-get install -y pcregrep
```

Comment:
Sorry, I got that wrong, I didn't need to install anything funny on Linux, but I did have to install GNU grep on Mac OS (otherewise you get BSD grep which doesn't have the `-P` option).

Here's the PR where I was making very similar changes to the same tests (but for different reasons).

https://github.com/cylc/cylc-flow/pull/5769/files